evap/student/tests/test_views.py

```diff
@@ -72,7 +68,7 @@ def test_incomplete_form(self):
         """
         page = self.get_assert_200(self.url, user=self.voting_user1.username)
         form = page.forms["student-vote-form"]
-        self.fill_form(form, False)
+        self.fill_form(form)
```

Comment:
sorry for being unclear, i meant specifying the parameter when calling the method (`self.fill_form(form, fill_complete=False)`), not in the method definition.